pkg/service/service.go

```diff
@@ -119,6 +121,14 @@ func (s *deviceService) Run() error {
 		},
 	})
 
+	// set poolSize to config.Device.AsyncBufferSize
+	config := container.ConfigurationFrom(s.dic.Get)
+	pool, err := ants.NewPool(config.Device.AsyncBufferSize)
```

Comment:
We reviewed the library and it looks good to be used.
However, I am just wondering how large capacity is the good default value.
In their document, they use 1000 or 10000 or example.
Did you use the default config.Device.AsyncBufferSize (16) to get the test result?  How many autoevent in your test?

Reply:
what is the number people ususally use for ants.NewPool?